zookeeper-server/src/test/java/org/apache/zookeeper/server/NettyServerCnxnTest.java

```diff
@@ -26,7 +26,14 @@
 import static org.junit.Assert.assertThat;
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import io.netty.channel.*;
```

Comment:
please don't do wild card imports - only import what's needed. (I am surprised that checkstyle does not catch this.).